pkg/autopilot/checks/checks.go

```diff
@@ -30,6 +30,9 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+//go:embed removed-gvks.yaml
```

Comment:
just a thought, how about writing that list as go consts?

Reply:
Yeah, was thinking the same, but I decided to leave this as a TODO for the future us, and focus on the mere technical transition away from go-bindata here.

Reply:
See #4916.